### PR TITLE
CI: Update the cloud image to latest

### PR DESCRIPTION
**Pull request by askb**
Also add the default volume size. The value is now
needed to be defined with the latest cloud provider
upgrades.

Signed-off-by: Anil Belur <abelur@linuxfoundation.org>

### CODE REVIEW

**Review by bthuree — APPROVED** [review]

**Comment by askb** [comment]
thanks @bthuree

File: jenkins-config/clouds/openstack/cattle/centos7-builder-1c-1g.cfg [hunk 1]
```diff
@@ -1,3 +1,4 @@
-IMAGE_NAME=ZZCI - CentOS 7 - builder - x86_64 - 20210519-032457.062
+IMAGE_NAME=ZZCI - CentOS 7 - builder - x86_64 - 20210519-041348.372
 VOLUME_SIZE=20
 HARDWARE_ID=v2-highcpu-1
+VOLUME_SIZE=20
```
Comment (eb-oss) [inline jenkins-config/clouds/openstack/cattle/centos7-builder-1c-1g.cfg:4]: Volume size is now listed twice. I think you should remove it from line 2.

**Comment by askb** [comment]
remerge

File: jenkins-config/clouds/openstack/cattle/centos7-builder-1c-1g.cfg (hunk 1 above)
Comment (askb) [reply jenkins-config/clouds/openstack/cattle/centos7-builder-1c-1g.cfg:4]: @MightyNerdEric This is fixed up here: https://github.com/zowe/ci-management/pull/29
Thanks for catching that.

**Pull request merged**